Dockerfile

```diff
@@ -1,28 +1,31 @@
-# specify the node base image with your desired version node:<version>
-FROM node:8-slim
+# Build stage
+FROM node:12.16.3-buster-slim AS umbrel-middleware-builder
```

Comment:
I wouldn't just use the latest LTS since Node.js can change a LOT between LTS versions, and dependencies sometimes take a while to get updated to work with the latest version.

You could trim it down to `node:12-buster-slim` though.

Reply:
Yeah, also `12.16.3-buster-slim` is consistent across `umbrel-manager` and `umbrel-dashboard` so can probably stick with it for now